pulsarsObjects.py

```diff
@@ -5,7 +5,7 @@
 import barcen
 
 # Loading the data
-def load_pulsar_data(pulsar_name, pulsarcat_file='small-data-files/pulsarcat.csv'):
+def load_pulsar_data(pulsar_name, pulsarcat_file='./../../small-data-files/pulsarcat.csv'):
```

Comment:
Zo werkt de code niet meer als hij vanaf een andere map wordt gerund. Kun je dat even fixen? Gewoon dirname van de python-file zelf opvragen in python. Iets als `Path(__file__).parent+'/small-data-files/pulsarcat.csv'`, en `from pathlib import Path` bovenin importeren.

Reply:
Of trouwens `Path(__file__).parent/'small-data-files/pulsarcat.csv'` is nog mooier :)

Reply:
Okay, ik zal er even naar kijken, dat is wel een goede suggestie. Mooiste is natuurlijk als het programma vanuit alle folders gerunt kan worden, het zelfde geldt dan voor de defaults.yml bestand.